util/QueryHelper.js

```diff
@@ -0,0 +1,40 @@
+class QueryHelper {
+  constructor(Model, query) {
+    this.Model = Model;
+    this.query = query;
+    this.qb = Model.query();
+    this.handlePagination();
+    this.handleSorting();
+    this.handleFields();
+  }
+
+  async handlePagination() {
```

Comment:
already supported http://vincit.github.io/objection.js/#paging

Reply:
Yeah, but you still need to remove the properties from the query. And you still need to chain onto the QB. We could use the page function instead of offset/limit though for sure.